sand/extensions/search/aggregated_search.py

```diff
@@ -0,0 +1,36 @@
+from typing import List, Union
+from sand.extension_interface.search import IEntitySearch, IOntologySearch
+from sand.models.search import SearchResult
+
+
+class AggregatedSearch(IEntitySearch, IOntologySearch):
+
+    def __init__(self):
+        self.search_entity_objs = []
+        self.search_ontology_objs = []
+
+    def add(self, search_obj: Union[IEntitySearch, IOntologySearch]):
+        """adds a search object to the aggregated search"""
+        self.search_entity_objs.append(search_obj)
```

Comment:
The reason I didn't add isinstance check because we are inheriting both IEntitySearch and IOntologySearch, it will return true all the time. which Is why I wanted have separate implementations.

Reply:
updated it.